README.md

```diff
@@ -59,24 +59,26 @@ a part of `site.yml`'s run. Use the `debug` option to get more insight to the pr
 
 Config file should be in: `/etc/ansible-puller/config.json`, `$HOME/.ansible-puller.json`, `./ansible-puller.json`
 
-| Config Option            | Default                               | Description                                                                             | Required |
```

Comment:
possible to not reformat everything?

Reply:
This actually removes the "Required" column because that column would be now blank